packages/theme/css/foundations/animation.css

```diff
@@ -21,10 +22,24 @@
   --uitk-animation-slide-out-left: slide-out-left var(--uitk-animation-duration) var(--uitk-animation-timing-function);
   --uitk-animation-slide-out-right: slide-out-right var(--uitk-animation-duration) var(--uitk-animation-timing-function);
   --uitk-animation-slide-out-bottom: slide-out-bottom var(--uitk-animation-duration) var(--uitk-animation-timing-function);
-  /* Fade in Animations */
+
+  /* Drawer Slide Animations */
```

Comment:
is there a reason to add this ones here instead of overriding `--uitk-animation-duration` in the layer layout css?

Reply:
@libertymayc suggested we move this into foundations to avoid having the override which makes sense if we ever have similar components that would use this duration instead of the default one